docs/index.md

```diff
@@ -1,6 +1,10 @@
 # lakefs-spec: An fsspec backend for lakeFS
 
-Welcome to lakeFS-spec, a [filesystem-spec](https://github.com/fsspec/filesystem_spec) backend implementation for the [lakeFS](https://lakefs.io/) data lake.
+!!! tip "Early Adopters"
+    We are seeking early adopters who would like to actively participate in our feedback process and shape the future of the library.
```

Comment:
WDYT about "We are seeking early adopters who we invite  to actively [...]"?

To not scare early adopters off by demanding too much work from them. On the other hand, this makes clear that we are available for help and their feedback can shape the tool.